pages/spicedb/modeling/composable-schemas.mdx

```diff
@@ -206,10 +203,4 @@ Attempting to reference other declaration types (e.g. a definition or a caveat)
 Then on merge:
 
 1. CI runs `zed preview schema compile`
-1. CI calls SpiceDB with the compiled schema
-
-## Notes
-
-SpiceDB `WriteSchema` calls cannot interpret `import` or `partial` syntax.
-If using the new syntax, you must build the schema using `compile` before it can be submitted.
-This is because a schema write must happen in a single call to SpiceDB.
```

Comment:
Feel like this is too important for it to be at the bottom, so I moved it to the top